energyplus_diffs/__main__.py

```diff
@@ -0,0 +1,4 @@
+from energyplus_diffs.cli import cli
+
+
+cli()
```

Comment:
New `python -m energyplus_diffs` entry point is just calling `cli()`, which is the exact same as pip installing the library and running the `energyplus_diffs` binary shim.  But some people like this better.  So here we are.